### ISSUE TITLE
Update labels on the Docker image

### ISSUE THREAD

**Issue by dliappis** [elastic/elasticsearch-docker#144]
Currently the Docker image merges labels from the base centos:7 image as well as our own [maintainer LABEL](https://github.com/elastic/elasticsearch-docker/blob/master/templates/Dockerfile.j2#L106):

```
$ docker inspect docker.elastic.co/elasticsearch/elasticsearch:6.1.1 | jq .[0].ContainerConfig.Labels
{
  "build-date": "20171128",
  "license": "GPLv2",
  "maintainer": "Elastic Docker Team <docker@elastic.co>",
  "name": "CentOS Base Image",
  "vendor": "CentOS"
}
```

We need to add more labels to clarify the license for Elastic products, as well as name/vendor/build-date.

**Comment by dliappis** [comment]
cc @mgreau @jarpy @beniwohli

**Comment by mgreau** [comment]
As I'm currently working on it for other Elastic projects right now, I think we should use [an SPDX identifier](https://spdx.org/licenses/) in the license label content.

Also, FYI a group is working on a convention for a standardized way to express container labels:
- http://label-schema.org/

It's still a WIP but it's maintained by well-known people in the container world:  https://github.com/label-schema/label-schema.org/blob/gh-pages/rc1.md

**Comment by jarpy** [comment]
It looks as though the Centos:7 image has recently moved to Label Schema but it [isn't quite right yet](https://github.com/CentOS/sig-cloud-instance-images/issues/114).

Anyway, we can be more confident that we can overwrite their labels, since they are clearly moving to Label Schema.

**Comment by jarpy** [comment]
Interestingly, Label Schema doesn't actually have a licence field.

**Comment by jarpy** [comment]
Fixed in #166
